Source/WebKit/Platform/cocoa/ExtensionCapabilityGrant.mm

```diff
@@ -51,15 +58,15 @@ static void platformInvalidate(id<BEProcessCapabilityGrant> platformGrant)
 {
 }
 
-ExtensionCapabilityGrant::ExtensionCapabilityGrant(String&& environmentIdentifier, RetainPtr<BEProcessCapabilityGrant>&& platformGrant)
+ExtensionCapabilityGrant::ExtensionCapabilityGrant(String&& environmentIdentifier, PlatformGrant&& platformGrant)
     : m_environmentIdentifier { WTFMove(environmentIdentifier) }
     , m_platformGrant { WTFMove(platformGrant) }
 {
 }
 
 ExtensionCapabilityGrant::~ExtensionCapabilityGrant()
 {
-    setPlatformGrant(nil);
+    platformInvalidate(m_platformGrant);
```

Comment:
Shouldn't this be `platformInvalidate(std::exchange(m_platformGrant, nullptr));`?